Review Section 3.8 Numbers

behnam opened the pull request:
Edit the section from beginning up to "3.8.4 Arabic number in other uses".

Tracker: <https://github.com/w3c/alreq/issues/101>

khaledhosny commented on index.html:
```diff
@@ -188,14 +188,14 @@
-      "uname">U+0669</span> and <span class="uname">U+06F0</span>–<span class=
-      "uname">U+06F9</span>) used by different languages. Most of the alphabetical characters are
+      <p>The majority of these characters are common among different languages. There are three
+      different set of digits used by different languages. Most of the alphabetical characters are
```
“different sets”.

behnam commented:
Thanks, @khaledhosny. Fixed the typo.

r12a commented on index.html:
```diff
@@ -188,14 +188,14 @@
 
-      <p>Arabic script uses Arabic alphabet, diacritics, numbers, punctuations and symbols, and
-      control characters. Appendix <a href="#characters-tables"></a> lists these characters.</p>
+      <p>Arabic script uses Arabic alphabet, which consists mostly of letters and diacritics,
```
I think better: The Arabic script uses the Arabic alphabet,

r12a replied:
Hmm, there are all sorts of problem here.

The start of the paragraph sounds  rather redundant, but i think it's actually incorrect. I don't think an alphabet includes digits and punctuation.  Furthermore, the 'Arabic alphabet' in my mind is different from the 'Persian alphabet', which is different from the Kashimiri or Uighur alphabets.  

Also, what does 'mostly' qualify?  Is is letters and diacritics only, or all the things in the list?  If the latter, what else does it consist of?

And then there's a mismatch between what this para says is in the table at the end and what's actually there, in that only two sets of digits are listed in the table.

Also, it's not just joining characters that are represented in the table.

How about simply:

Appendix ... lists characters used for the Arabic and Persian languages. Characters used for these languages include letters and diacritics, three sets of digits (depending on the region), punctuation (some common and some specific to the script), symbols, and Unicode formatting characters.

behnam replied:
Agreed. Updated the text based on your comment.

r12a commented:
@behnam I'm not sure why you felt the need to change the ids from h_xxx to xxx. As a consequence, all the links to the document from issues and from the text layout index (eg. http://w3c.github.io/typography/#glyphs_diacritics), etc., will now break and need to be fixed. I don't think that's a good thing, and i don't really want to take the time needed to edit the layout index doc just for that.

For the figure ids (and ids in general), i recommend that you don't use hyphen-minus. figure_id can be copied to the clipboard with a single doubleclick, but for figure-id you need to select and extend the range (which has to be done any time you want to link to that figure).  It's a small thing, but it mounts up.

behnam committed:
Review Section 3.8 Numbers

Edit the section from beginning up to "3.8.4 Arabic number in other
uses".

Tracker: <https://github.com/w3c/alreq/issues/101>

behnam commented:
Okay, dropped the changes on ids from this PR, so we can go ahead with this while discussing that.
